boost-common/src/main/java/io/openliberty/boost/utils/ConfigConstants.java

```diff
@@ -12,10 +12,22 @@
 package io.openliberty.boost.utils;
 
 public interface ConfigConstants {
-    public String FEATURE = "feature";
-    public String FEATURE_MANAGER = "featureManager";
-    public String HTTP_ENDPOINT = "httpEndpoint";
-    public String DEFAULT_HTTP_ENDPOINT = "defaultHttpEndpoint";
+
+	public String FEATURE = "feature";
+	public String DEPENDENCY = "dependency";
+	public String GROUPID = "groupId";
+	public String ARTIFACTID = "artifactId";
+	public String VERSION = "version";
+	public String WAR_PKG_TYPE = "war";
+	
+	public String SPRING_BOOT_PROJ = "spring-boot-project";
+	public String EE_APP_PROJ = "project";		
```

Comment:
Maybe don't name this `EE_APP_PROJ`. The app can still be a Spring project, it just doesn't use the `springBootUtility`. I would suggest just naming it `NORMAL_PROJ`.